spec/factories/email_links.rb

```diff
@@ -0,0 +1,14 @@
+FactoryGirl.define do
+  factory :email_link do
+     email_subject "cray computer"
+     from_email "kegreen22@gmail.com"
+     from_name "K Green"
+     url "www.nytimes.com"
+     title "Cray invents new computer"
+     accept_or_rejected_at nil
+     accepted nil
+     read nil
+     end
```

Comment:
looks like a small indenting error here.

Also, take a look at the [Factory Girl documentation](http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md) regarding traits. Above, you're building an EmailLink object with a bunch of attributes specified (accepted: true ... ) and it could be more clear what your intent is for building the object in that way.

Here, you could add two traits like

```ruby
  trait :accepted_to_read do
    accepted: true
    accepted_or_rejected_at Time.now
  end

  trait :unread do
    read false
  end

  factory :accepted_and_unread_email_link, traits: [:accepted, :unread]
```

And then update your controller spec with this new factory. This has the added benefit that if the business logic for these objects changes, we just need to update this factory.